docs/en/latest/admin-api.md

```diff
@@ -277,6 +277,10 @@ curl 'http://127.0.0.1:9180/apisix/admin/routes?name=test&uri=foo&label=' \
 
 Route resource request address: /apisix/admin/routes/{id}?ttl=0
 
+### Quick Note on ID Syntax
+
+ID's as a text string must be of a length between 1 and 64 characters and they should only contain uppercase, lowercase, numbers and no special characters apart from dashes ( - ) and underscores ( _ ). For integer values they simply must have a minimum character count of 1.
```

Comment:
It also accepts periods ".". Please include this as well.

Reply:
I've included that too. Should I squash the commits?